parameter_files/ProjectParameters_PA2024CH.yml

```diff
@@ -0,0 +1,74 @@
+default:
+
+    paths:
+        data_location_ext: ../pacta-data/2023Q4/
+
+    reporting:
+        project_report_name: PA2024CH
+        display_currency: CHF
+        currency_exchange_value: 1.190405
+
+    parameters:
+        timestamp: 2023Q4
+        dataprep_timestamp: 2023Q4
```

Comment:
I'm pretty sure these are never used anymore

Reply:
These get read in by `pacta.portfolio.utils::set_project_parameters()` as `financial_timestamp` and `dataprep_timestamp` here:
https://github.com/RMI-PACTA/pacta.portfolio.utils/blob/6f2158548b552fa4204179eb5bf49974f3a93ccf/R/set_project_parameters.R#L20-L21

`dataprep_timestamp` was deprecated in #216 

`financial_timestamp` seems to have never been used in this repo:
https://github.com/search?q=repo%3ARMI-PACTA%2Fworkflow.transition.monitor%20financial_timestamp&type=code

so I'm going to remove these